server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/IdentitySetContract.scala

```diff
@@ -91,24 +99,65 @@ trait IdentitySetContract {
       .asString
 
     assertThatJson(response)
-      .inPath("methodResponses[0][1]")
+      .whenIgnoringPaths("methodResponses[0][1].created.4f29.id", "methodResponses[1][1].list[0].id")
```

Comment:
This ignoreing works but it looks a little ugly. I will try to get the actual id tomorrow :')

Reply:
There's a way with $$ignore to bypass a value with json asserts on value strings